src/test/java/org/example/tests/MultiItemLogoutTest.java

```diff
@@ -0,0 +1,33 @@
+package org.example.tests;
+
+import org.example.enums.UserCredentials;
+import org.example.pages.LoginPageBase;
+import org.example.pages.MenuPageBase;
+import org.example.pages.ProductPageBase;
+import org.example.utils.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MultiItemLogoutTest extends BaseTest {
+
+    @Test
+    public void testAddMultipleItemsThenLogout() {
```

Comment:
Separate logout test in this case might be redundant if you already have a dedicated test for the logout process. 
 Since you already have a test (testLogoutFlow) that verifies the logout process independently, you don't need to duplicate this functionality in a test where the main focus is on adding items to the cart and performing the logout.